README.md

```diff
@@ -4,6 +4,9 @@
 
 ![Flyway](https://flywaydb.org/assets/logo/flyway-logo-tm.png "Flyway")
 
+#### Changes log
+* Support GreenPlum 4.3.7.1  Database 
+
```

Comment:
No new sections please, just add this to the regular list of databases. Version details go on flywaydb.org